spacy_llm/tasks/spancat/task.py

```diff
@@ -0,0 +1,112 @@
+from typing import Any, Callable, Dict, Iterable, List, Optional, Type
+
+from spacy.language import Language
+from spacy.tokens import Doc, Span
+from spacy.training import Example
+
+from ...compat import Literal, Self
+from ...ty import Scorer, TaskResponseParser
+from ..span import SpanExample, SpanTask
+from ..templates import read_template
+
+DEFAULT_SPANCAT_TEMPLATE_V1 = read_template("spancat.v1")
+DEFAULT_SPANCAT_TEMPLATE_V2 = read_template("spancat.v2")
+
+
+class SpanCatTask(SpanTask):
+    def __init__(
+        self,
+        parse_responses: TaskResponseParser[Self],
+        prompt_example_type: Type[SpanExample],
+        labels: List[str],
+        template: str,
+        label_definitions: Optional[Dict[str, str]],
+        spans_key: str,
+        prompt_examples: Optional[List[SpanExample]],
+        normalizer: Optional[Callable[[str], str]],
+        alignment_mode: Literal["strict", "contract", "expand"],
+        case_sensitive_matching: bool,
+        single_match: bool,
+        scorer: Scorer,
+    ):
+        """Default SpanCat task.
+
+        parse_responses (TaskResponseParser): Callable for parsing LLM responses for this task.
+        fewshot_example_type (Type[FewshotExample]): Type to use for fewshot examples.
```

Comment:
I think there's one inconsistency left here after all the renaming back-and-forth: `prompt_example_type` vs. `fewshot_example_type` (throughout the PR)

Reply:
Let's stick with `prompt_example_type` for now?

Reply:
Updated in https://github.com/explosion/spacy-llm/pull/242/commits/b804c5b48bf8b69f1271844ea77d88537a6bc07f.